### PR TITLE
Conditions

### PR DESCRIPTION
**Pull request by Maxscores**
Solution for seed error. Seed was successful. Had to put some error catching in. It works!

### CODE REVIEW

**Comment by Maxscores** [comment]
Hey guys, make sure to check out the error messages I was getting on this. It is pretty interesting.

**Comment by Maxscores** [comment]
Got the first condition breakdown!!!!!!!!!!!

File: db/seed.rb [hunk 1]
```diff
@@ -61,21 +62,25 @@ def self.trips
         zipcode: row[:zip_code],
         condition_id: Condition.find_by(date: Date.strptime(row[:start_date], '%m/%e/%Y')).id
       )
-
+    rescue
+      puts "Caught Seed Error"
+      end
     end
```
Comment (Ecksi) [inline db/seed.rb:68]: Cool use of begin / rescue. What made you think about using these?

File: db/seed.rb [hunk 2]
```diff
@@ -61,21 +62,25 @@ def self.trips
         zipcode: row[:zip_code],
         condition_id: Condition.find_by(date: Date.strptime(row[:start_date], '%m/%e/%Y')).id
       )
-
+    rescue
+      puts "Caught Seed Error"
```
Comment (Ecksi) [inline db/seed.rb:66]: If we have to use a lot of these rescues (maybe where we are having errors in Station index) could we turn this into a method to call on?

**Review by Ecksi — APPROVED** [review]

**Pull request merged**